roles/falcon_configure/README.md

```diff
@@ -100,10 +105,22 @@ Ensure the following API scopes are enabled (***if applicable***) for this role:
 - When using API credentials `falcon_client_id` and `falcon_client_secret`
   - **Sensor Download** [read]
 
+## Falcon Sensor Service
+
+For Linux/macOS, the Falcon Sensor service is automatically started after the sensor is installed (*although it will be in a failed state*).
+
+> [!WARNING]
+> The service will be in a failed state until the CID is set and the sensor is restarted.
+
+This role will restart the Falcon Sensor service after the CID is set, along with any other configuration options.
+
```

Comment:
This is almost more confusing than saying nothing. What is the point you're trying to get across?

Reply:
The point I'm trying to get across is that folks are confused by how the falcon-sensor is started.. the thing is, the sensor is always started by default, it's just in a failed state until you provide the CID and restart the sensor. It seems like people try to manage the falcon-sensor service manually since it's apparently not clear in here.

Any thoughts on properly wording that?